### ISSUE TITLE
installApp task has been deprecated

### ISSUE THREAD

**Issue by ErichBSchulz** [cqframework/clinical_quality_language#55]
not sure if this is a problem for you - but just an fyi that my successful (yay!) build complained thusly:

```
The installApp task has been deprecated and is scheduled to be removed in Gradle 3.0. Please use       
the installDist task instead.
```

**Comment by cmoesel** [comment]
Thanks.  I don't think our use of `installApp` is central to any of the libraries, but it can be a convenient way of building a distributable package for each of our commandline utilities.  Switching to `installDist` shouldn't be hard -- so it should definitely go on the list!

**Comment by duncand** [comment]
I confirm that this issue still occurs on 2017 Sep 5 with the latest versions of everything.

**Comment by brynrhodes** [comment]
This issue was addressed by the fix for #247.
